whelk-core/src/main/resources/ext/marcframe.md

```diff
@@ -0,0 +1,203 @@
+# MARCFRAME
+
+> The purpose of this document is for developers and others who need to understand the mechanisms of MARCFRAME, the configuration file which handles conversion of MARC21 to JSON-LD and the roundtrip JSON-LD to MARC21. The context is the Libris implementation of MARC21.
+
+## Structure
+
+Marcframe is comprised of mainly three sections which represents three different MARC21 formats, bibliographic, authority and holdings. In addition there is an introductory part which contains reusable patterns mappings of codes to terms.
+
+
+```
+{
+  {bib},
+  {auth},
+  {hold}
+}
+```
+
+For each section there is a chronological mapping of the fields 000-999 and their subfields to properties and classes which are definied either in the kbv: vocabulary or the group of terms which get a marc: prefix. Marc terms are used where the conversion do not fit in a Bibframe 2 form or the usage has in other ways been ambiguous.
+
+The most common reason for a marc-prefix is LC combining two fields so it is not possible to separate the data making a reconversion impossible, or that they haven't made any attempts to code the information either with nac - no attempt to code, or ignore for things which have no meaning outside MARC21.
+
+There are also certain postprocessing steps which can handle particular fields, for example date/year which needs to be exported to several fields (E.g. 008 + 264) depending on how they are formatted.
+
+## Syntax in MARCFRAME
+
+This is a list of keys in Marcframe which control certain functionality like repetition, grouping, inclusion of patterns etc. 
+
+`about` enables shortforms of more complicated or nested entitites.
+
+`aboutAlias` used to include an about form.
+
+`aboutEntity` Defines which entity the property is a part of, **thing** means instance in bibliographic data.
+
+`aboutType` -
```

Comment:
Missing description?

Reply:
Ah yes, @niklasl could you provide something comprehensible? :)